hackathon_site/dashboard/frontend/src/components/general/TitledPaper/TitledPaper.test.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import TitledPaper from "./TitledPaper";
+
+describe("<TitledPaper />", () => {
+    const titleTest = "Test title";
+    const childTest = "Test child";
+    it("matches snapshot", () => {
```

Comment:
Imo, this test isn't really necessary. I think in general we should move away from snapshot tests of components like this, since it doesn't add much value. Thoughts?